core-bundle/assets/scripts/core.js

```diff
@@ -1037,11 +1022,9 @@ window.Backend =
 					if (droppable.hasClass('tl_folder') && currentHover !== droppable) {
 						currentHover = droppable;
 						currentHoverTime = new Date().getTime();
+						expandLink = droppable.getElement('a.foldable');
 
-						var expandLink = droppable.getElement('img[src$="/icons/folPlus.svg"]');
-						expandLink = expandLink && expandLink.getParent('a');
-
-						if (expandLink) {
+						if (expandLink && !expandLink.hasClass('foldable--open')) {
```

Comment:
@ausi I now use the `.foldable` class instead of the image source to find the expand link, which worked in my tests. Any objections?

Reply:
No objections, seems fine to me :+1: